mbuild/formats/charmm_writer.py

```diff
@@ -1049,6 +1049,59 @@ def _lengths_angles_to_vectors(lengths, angles, precision=6):
     return box_vectors
 
 
+def check_fixed_lists(
+    fixed_residues_list, fixed_residues_variable_name, residues
+):
+    """Check the GOMC fixed bonds and angles lists for input errors.
+
+    Parameters
+    ----------
+    fixed_residues_list : list of strings, [str, ..., str]
+        A list of the residues to be fixed for the GOMC simulation engine.
+        In GOMC, the residues currently the molecules.
```

Comment:
Looks like a typo here. Not sure what this is supposed to mean

Reply:
I reworked this, let me know if it makes more sense now.

    fixed_residues_list : list of strings, [str, ..., str]
        A list of the residues (i.e., molecules since GOMC currently considers a
        a whole molecule as a residue) to have their bonds and/or angles held
        rigid/fixed for the GOMC simulation engine.
        In GOMC, the residues currently the molecules.